gallery/src/components/NxCombobox/NxComboboxPage.tsx

```diff
@@ -0,0 +1,268 @@
+              <NxTable.Cell></NxTable.Cell>
+              <NxTable.Cell>
+                Validation failure messages.
+              </NxTable.Cell>
+            </NxTable.Row>
+            <NxTable.Row>
+              <NxTable.Cell>HTML <NxCode>&lt;div&gt;</NxCode> Attributes</NxTable.Cell>
+              <NxTable.Cell>
+                <NxTextLink external href="https://developer.mozilla.org/en/docs/Web/HTML/Element/div">
+                  Div Attributes
+                </NxTextLink>
+              </NxTable.Cell>
+              <NxTable.Cell>No</NxTable.Cell>
+              <NxTable.Cell></NxTable.Cell>
+              <NxTable.Cell>
+                <NxCode>NxCombobox</NxCode> supports any HTML attribute that's normally supported by{' '}
+                HTML <NxCode>&lt;div&gt;</NxCode>.
+              </NxTable.Cell>
+            </NxTable.Row>
+          </NxTable.Body>
+        </NxTable>
+      </NxTile.Subsection>
+      <NxTile.Subsection>
+        <NxTile.SubsectionHeader>
+          <NxH3>NxCombobox CSS Classes</NxH3>
+        </NxTile.SubsectionHeader>
+        <NxTable>
+          <NxTable.Head>
+            <NxTable.Row>
+              <NxTable.Cell>Class</NxTable.Cell>
+              <NxTable.Cell>Location</NxTable.Cell>
+              <NxTable.Cell>Details</NxTable.Cell>
+            </NxTable.Row>
+          </NxTable.Head>
+          <NxTable.Body>
+            <NxTable.Row>
+              <NxTable.Cell><NxCode>nx-combobox--short</NxCode></NxTable.Cell>
+              <NxTable.Cell>
+                Directly on the <NxCode>NxCombobox</NxCode>
+              </NxTable.Cell>
+              <NxTable.Cell>Short variant of the <NxCode>NxCombobox</NxCode>.</NxTable.Cell>
+            </NxTable.Row>
+            <NxTable.Row>
+              <NxTable.Cell><NxCode>nx-combobox--long</NxCode></NxTable.Cell>
+              <NxTable.Cell>
+                Directly on the <NxCode>NxCombobox</NxCode>
+              </NxTable.Cell>
+              <NxTable.Cell>Long variant of the <NxCode>NxCombobox</NxCode>.</NxTable.Cell>
+            </NxTable.Row>
+          </NxTable.Body>
+        </NxTable>
+      </NxTile.Subsection>
+      <NxTile.Subsection>
+        <NxTile.SubsectionHeader>
+          <NxH3>Helpers</NxH3>
+        </NxTile.SubsectionHeader>
+        <NxP>
+          If using <NxCode>NxCombobox</NxCode> for a backend query, the search functionality should be used with a
+          debounce in order to prevent excessive queries to the backend. The standard timing value to use for that
+          debounce is provided via the <NxCode>NX_STANDARD_DEBOUNCE_TIME</NxCode> export.
+        </NxP>
+        <NxWarningAlert>
+          <NxCode>NX_SEARCH_DROPDOWN_DEBOUNCE_TIME</NxCode> has been renamed to{' '}
+          <NxCode>NX_STANDARD_DEBOUNCE_TIME</NxCode>, it is now deprecated but is still exported by RSC for
+          backwards compatibility.
+        </NxWarningAlert>
+      </NxTile.Subsection>
```

Comment:
`NxSearchDropdown` has a "Usage Notes" section which I think it equally applicable to `NxCombobox` and should be copied here.

Reply:
Updated here c88f1abdb10667054bc42ea3d8ef210e6d81eaf1.